openstack/designate/templates/etc/_designate_audit_map.yaml

```diff
@@ -0,0 +1,23 @@
+service_type: 'dns'
+service_name: 'designate'
+
+prefix: '/v2.0'
+
+resources:
+  zones:
+    custom_actions:
+      
+  pools:
+  limits:
+    singleton: true
+  # Default mappings
+  tlds:
+  tsigkeys:
+  blacklists:
+  quotas:
+    singleton: true
+
+  service_statuses:
+    service_name: 'name'
```

Comment:
why service_name? the service_name is always the same. Only the resource typeURI varies.

Reply:
do we have a resource "service_statuses" with multiple instances? the typeURI of the elements would be 'service_statuse' unless you override it.

Reply:
/v2/service_status is a singleton that returns the health state of services. perhaps it shouldn't be included?